atlas/package.json

```diff
@@ -7,11 +7,11 @@
 	"scripts": {
 		"start": "concurrently \"yarn dev\" \"yarn storybook\"",
 		"build": "lerna run build",
-		"dev": "lerna run --scope atlas-app --stream dev",
+		"dev": "lerna run --scope atlas-app dev",
 		"storybook": "lerna run --scope atlas-app --stream storybook",
 		"lint": "yarn run eslint . --ext .js,.jsx,.ts,.tsx",
 		"clean": "lerna clean && rm -rf node_modules",
-		"test": "jest"
+		"test": "lerna run --stream test"
```

Comment:
I suggest we change this to `yarn workspace atlas-app test` since that gives a nicer output and allows us to pass jest CLI options like `--coverage` directly, which currently doesn't work with lerna.
Otherwise LGTM!

Reply:
Good idea, I've also used the same approach to fix `yarn dev` from the root